client-participation/gulpfile.js

```diff
@@ -232,14 +233,14 @@ gulp.task("connect", [], function () {
   app.use(/^\/wimp$/, express.static(path.join(destRootBase, "wimp.html")));
   app.use(/^\/try$/, express.static(path.join(destRootBase, "try.html")));
 
-  app.listen(polisConfig.PORT);
-  console.log("listening on localhost:" + polisConfig.PORT);
+  app.listen(polis_config.get('port'));
+  console.log("listening on localhost:" + polis_config.get('port'));
 });
 
 function getGitHash() {
   return new Promise(function (resolve, reject) {
-    if (process.env.GIT_HASH) {
-      resolve(process.env.GIT_HASH);
+    if (polis_config.get('git_hash')) {
```

Comment:
It's _possible_ that `GIT_HASH` is one of the few config options we'd actually want to _keep_ as an environment variable. IIRC, the way it's currently used is that it gets set from one of the scripts (gulpfiles, etc) when you want to deploy a specific ref. We haven't really been using that so much though, and those pathways may need to be updated to get it working again, given other changes we've been making to deployment. Something to look into an verify.

Reply:
Nevermind; Missed context from post that config settings can still be set from environment variables, so this is probably fine.

Still may be worth reviewing how `GIT_HASH` was used, and whether anything else needs to update in relation to it.